### PR TITLE
Initdb attachments

### LINKED ISSUE #6: click-odoo-initdb must store attachments in database when cache is enabled
Otherwise databases created from cached templates are inconsistent.

### PR DESCRIPTION
**Pull request by sbidoul**
Fixes #6

### CODE REVIEW

File: click_odoo_contrib/initdb.py [hunk 1]
```diff
@@ -35,7 +35,21 @@ def check_cache_prefix(cache_prefix):
             "Invalid cache prefix name '{}'".format(cache_prefix))
 
 
-def odoo_createdb(dbname, demo, module_names):
+def _patch_ir_attachment_store():
+
+    def _db_storage(self):
+        return 'db'
+
+    from odoo.addons.base.ir.ir_attachment import IrAttachment
```
Comment (ThomasBinsfeld) [inline click_odoo_contrib/initdb.py]: @sbidoul I get an error here (9.0). And it works if i replace `odoo` by `openerp`.

```
ImportError: No module named addons.base.ir.ir_attachment
```

File: click_odoo_contrib/initdb.py [hunk 2]
```diff
@@ -35,7 +35,21 @@ def check_cache_prefix(cache_prefix):
             "Invalid cache prefix name '{}'".format(cache_prefix))
 
 
-def odoo_createdb(dbname, demo, module_names):
+def _patch_ir_attachment_store():
+
+    def _db_storage(self):
```
Comment (ThomasBinsfeld) [inline click_odoo_contrib/initdb.py]: @sbidoul I had to write it with old api to make it work on 9.0.

```python
@odoo.api.v7
def _db_storage(self, cr, uid, context=None):
    return 'db'
```

Comment (sbidoul) [reply click_odoo_contrib/initdb.py]: The good thing is that the test suite caught it.

**Comment by sbidoul** [comment]
Merged manually at 3d1b6fbd7116925e67d7c13c8b4d2d9927f8bf16

**Pull request closed without merging**